kuma/install.go

```diff
@@ -61,19 +64,20 @@ func (kuma *Kuma) installUsingManifests(del bool, st string, namespace string, v
 	}
 	return status.Installed, nil
 }
-func (kuma *Kuma) applyHelmChart(del bool, version, namespace string) error {
+func (kuma *Kuma) applyHelmChart(del bool, namespace string) error {
 	kClient := kuma.MesheryKubeclient
 	if kClient == nil {
 		return ErrNilClient
 	}
 	kuma.Log.Info("Installing using helm charts...")
+	version := getLatestKumaChartVersion()
+	url := "https://github.com/kumahq/charts/releases/download/" + version + "/" + version + ".tgz"
 	err := kClient.ApplyHelmChart(mesherykube.ApplyHelmChartConfig{
 		ChartLocation: mesherykube.HelmChartLocation{
-			Repository: "https://kumahq.github.io/kuma",
```

Comment:
Hmm I see you had to move away from using their helm repo here...
My bad. I pointed you to the wrong repo. I see now that the chart stored here is way too old.
However, https://kumahq.github.io/charts/ seems to be up to date. We can use this and let meshkit handle the versioning internally.

Reply:
@DelusionalOptimist  this repo link works fine when I dont pass in AppVersion. But when I pass AppVersion as version passed in argument, the installation fails.

Reply:
I checked then found out , the version that was being passed was 1.2.2 and this is indeed present as appversion in index.yaml in the helm repo. So I am tryna figure out, where is the issue.

Reply:
Found the normalization issue and pointed out in comment above.